reproducibility_FAQs.md

```diff
@@ -0,0 +1,253 @@
+
+### How do I check if my code is stable?
+**Stable code**  produces the same outputs every time it is run with the same inputs.  **Unstable code**  yields different results on repeated runs, which undermines reproducibility.
+
+**How to test for stability:**  
+Run the full code twice. If any outputs differ between runs, the code is unstable.
+
+**How to track output changes:**
+To facilitate version control and comparison:
+-   Export **figures** in standardized formats such as `.png` or `.jpg` 
+-   Save  **tables**  in  `.csv`,  `.txt`, or  `.tex`
+-   Use  **Git**  to version-control your outputs and detect changes across runs
+    
+
+**Common causes of instability:**
+
+-   Sorting on non-unique ID variables before creating or dropping observations/variables
+-   Not setting a seed for random number generation
+-   Using different versions of the Statistical software
+-   Using  `m:m`  merges
+-   Running different versions of user-written packages
+    
+Ensuring code stability is a key step in creating reproducible research.
+
+### How do I set up a reproducible environment?
+
+A  **reproducible environment**  ensures that anyone running your code has the same setup: software versions, packages, and dependencies, so the results remain consistent over time and across machines.
+
+To set this up, follow the environment setup guides for each language:
+-   **Python Environment Setup:**  [Python Instructions](https://worldbank.github.io/wb-reproducible-research-repository/resources/environment-instructions/python.html)
+-   **R Environment Setup:**  [R Instructions](https://worldbank.github.io/wb-reproducible-research-repository/resources/environment-instructions/R.html)
+-   **Stata Environment Setup:**  [Stata Instructions](https://worldbank.github.io/wb-reproducible-research-repository/resources/environment-instructions/stata.html)
+
+Using a clean, well-defined environment is critical to computational reproducibility.
+
+### Do you check if the code is correct?
+No. We only verify complete documentation and **computational reproducibility**: whether running the same code and data produces the same outputs.  We do not assess whether the code correctly implements the intended methods, nor the quality of the code.
+
+## Documentation-Related Questions
+
+### What is a README and what should it include?
+
+A  **README**  is the main guide to your reproducibility package. It helps others—including reviewers, editors, and future researchers—understand how to navigate, run, and evaluate your code and data. A clear and complete README is essential for transparency and reproducibility. A recommended template is available [here](https://github.com/worldbank/wb-reproducible-research-repository/blob/main/resources/README_Template.md).
```

Comment:
suggest adding that if the publication is a journal article they should use the social science data editors template. mention our template is a simplified version of that.